### ISSUE TITLE
maketx for single channel dwaa 

### ISSUE THREAD

**Issue by ThiagoIze** [OpenImageIO/oiio#1844]
When running maketx on a single channel image and specifying exr dwaa output, I get a fail.

To repro (tested on latest master):
```
% oiiotool --pattern checker 512x512 1 -o test.jpg
% maketx --format exr --compression dwaa test.jpg
maketx ERROR writing "test.8d714779.temp.tx" : Failed OpenEXR write: Failed to write pixel data to image file "test.8d714779.temp.tx". Data compression (zlib) failed.
```
But, if I create the dwaa exr directly through oiiotool then it seems to work:
```
% oiiotool --pattern checker 512x512 1 --compression dwaa -o test.exr
% iinfo -v test.exr
test.exr :  512 x  512, 1 channel, float openexr
    channel list: Y
    oiio:ColorSpace: "Linear"
    compression: "dwaa"
...
```

**Comment by lgritz** [comment]
Accept another apology from me for letting this issue fall through the cracks.

After some investigation, it seems that any attempt to write tiled exr with dwaa compression will hit this error if the tile size is < 16x16. 

In your original example, the reason it failed for maketx but not the oiiotool command is that the maketx, in writing the MIP levels, eventually gets down to 8x8 resolution and that tile size fails.

Here's an even simpler repro case that doesn't involve MIP maps:

    # works
    oiiotool --pattern checker 512x512 1 -d half -compression dwaa -tile 16 16 -o test.exr
    
    # fails
    oiiotool --pattern checker 512x512 1 -d half -compression dwaa -tile 8 8 -o test.exr

Sure seems like an OpenEXR internals bug. I will report to them, I don't think there's much I can do on my side.

**Comment by lgritz** [comment]
Closing. Fixed by #2147.
